paasta_tools/cli/cmds/validate.py

```diff
@@ -598,7 +632,7 @@ def validate_secrets(service_path):
     return return_value
 
 
-def paasta_validate_soa_configs(service, service_path):
+def paasta_validate_soa_configs(service, service_path, verbose=False):
```

Comment:
while we're here, we can be good scouts and leave this a better place than we found it and add some types:
```suggestion
def paasta_validate_soa_configs(service: str, service_path: str, verbose: bool = False) -> bool:
```